Data deployment fails

https://github.com/PolicyEngine/policyengine-us-data/actions/runs/12751801169

Build and test failing with
>make: *** wait: No child processes.  Stop.
make: *** Waiting for unfinished jobs....
make: *** wait: No child processes.  Stop.
Error: Process completed with exit code 143.

Docker failing around HDF5:

> #8 4.816 Collecting tables==3.9.2 (from policyengine-us)
> #8 4.835   Downloading tables-3.9.2.tar.gz (4.7 MB)
> #8 4.922      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.7/4.7 MB 65.4 MB/s eta 0:00:00
> #8 5.212   Installing build dependencies: started
> #8 11.39   Installing build dependencies: finished with status 'done'
> #8 11.39   Getting requirements to build wheel: started
> #8 12.79   Getting requirements to build wheel: finished with status 'error'
> #8 12.79   error: subprocess-exited-with-error
> #8 12.79   
> #8 12.79   × Getting requirements to build wheel did not run successfully.
> #8 12.79   │ exit code: 1
> #8 12.79   ╰─> [11 lines of output]
> #8 12.79       /tmp/H5closel4o4aw47.c:1:10: fatal error: H5public.h: No such file or directory
> #8 12.79           1 | #include "H5public.h"
> #8 12.79             |          ^~~~~~~~~~~~
> #8 12.79       compilation terminated.
> #8 12.79       * Using Python 3.13.1 (main, Dec 25 2024, 01:33:28) [GCC 12.2.0]
> #8 12.79       * Found cython 3.0.11
> #8 12.79       * USE_PKGCONFIG: True
> #8 12.79       .. ERROR:: Could not find a local HDF5 installation.
> #8 12.79          You may need to explicitly state where your local HDF5 headers and
> #8 12.79          library can be found by setting the ``HDF5_DIR`` environment
> #8 12.79          variable or by using the ``--hdf5`` command-line option.
> #8 12.79       [end of output]
> #8 12.79   
> #8 12.79   note: This error originates from a subprocess, and is likely not a problem with pip.
> #8 12.93 error: subprocess-exited-with-error
> #8 12.93 
> #8 12.93 × Getting requirements to build wheel did not run successfully.
> #8 12.93 │ exit code: 1
> #8 12.93 ╰─> See above for output.
> #8 12.93 
> #8 12.93 note: This error originates from a subprocess, and is likely not a problem with pip.
> #8 13.19 make: *** [Makefile:12: install] Error 1
> #8 ERROR: process "/bin/sh -c make install" did not complete successfully: exit code: 2
> ------
>  > [3/3] RUN make install:
> 12.79   
> 12.79   note: This error originates from a subprocess, and is likely not a problem with pip.
> 12.93 error: subprocess-exited-with-error
> 12.93 
> 12.93 × Getting requirements to build wheel did not run successfully.
> 12.93 │ exit code: 1
> 12.93 ╰─> See above for output.
> 12.93 
> 12.93 note: This error originates from a subprocess, and is likely not a problem with pip.
> 13.19 make: *** [Makefile:12: install] Error 1
> ------
> policyengine_us_data.Dockerfile:3
> --------------------
>    1 |     FROM python:latest
>    2 |     COPY . .
>    3 | >>> RUN make install
>    4 |     
> --------------------
> ERROR: failed to solve: process "/bin/sh -c make install" did not complete successfully: exit code: 2
> Error: Process completed with exit code 1.

--- a/.github/workflows/pull_request.yaml
+++ b/.github/workflows/pull_request.yaml
@@ -49,7 +49,7 @@ jobs:
         run: ".github/is-version-number-acceptable.sh"
   test:
     name: Build and test
-    runs-on: ubuntu-latest
+    runs-on: larger-runner
     steps:
       - name: Checkout code
         uses: actions/checkout@v4

--- a/.github/workflows/push.yaml
+++ b/.github/workflows/push.yaml
@@ -26,7 +26,7 @@ jobs:
         run: black . -l 79 --check
   test:
     name: Build and test
-    runs-on: ubuntu-latest
+    runs-on: larger-runner
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
     steps:
       - name: Checkout code

--- a/changelog_entry.yaml
+++ b/changelog_entry.yaml
@@ -1,0 +1,4 @@
+- bump: patch
+  changes:
+    fixed:
+    - Minor bug with memory breaches.

